hasher.go

```diff
@@ -193,11 +206,7 @@ func (n *Hasher) HashNode(l, r []byte) []byte {
 
 	// Note this seems a little faster than calling several Write()s on the
 	// underlying Hash function (see: https://github.com/google/trillian/pull/1503):
-	data := append(append(append(
-		make([]byte, 0, 1+len(l)+len(r)),
-		NodePrefix),
-		l...),
-		r...)
+	data := append(append(append(make([]byte, 0, 1+len(left)+len(right)), NodePrefix), left...), right...)
```

Comment:
[optional] I find this a lot more readable but it's subjective

```golang
	data := make([]byte, 0, 1+len(left)+len(right))
	data = append(data, NodePrefix)
	data = append(data, left...)
	data = append(data, right...)
```

Reply:
Agree!